module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderContentCard.kt

```diff
@@ -14,18 +14,27 @@ import org.cru.godtools.shared.renderer.state.State
 import org.cru.godtools.shared.tool.parser.model.Card
 
 @Composable
-internal fun ColumnScope.RenderContentCard(
+internal fun RenderContentCard(
     card: Card,
     state: State,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(10.dp)
 ) {
-    Column(
-        modifier
-            .background(card.backgroundColor.toComposeColor())
+    ElevatedCard(
+        colors = CardDefaults.cardColors(
+            containerColor = card.backgroundColor.toComposeColor(),
+        ),
+        elevation = CardDefaults.cardElevation(
+            defaultElevation = 6.dp
+        ),
+        modifier = modifier
             .fillMaxWidth()
             .padding(contentPadding)
     ) {
-        RenderContent(card.content, state)
+        Column(
+            modifier = modifier
```

Comment:
you shouldn't re-use the modifier inside the Card itself.

The Modifier parameter provides customization for the base view that the composable produces, if you reuse it in inner views consumers will get weird results. e.g. if a consumer wants to add 50dp of padding around the card they could use a passed in Modifier for that, by reusing it in the card content it will also add 50dp of padding between the card and card content